packages/fxa-settings/src/components/Ready/index.tsx

```diff
@@ -65,44 +68,72 @@ const Ready = ({
   isSignedIn = true,
   serviceName,
   viewName,
+  isSync = false,
+  errorMessage,
 }: ReadyProps & RouteComponentProps) => {
   usePageViewEvent(viewName, REACT_ENTRYPOINT);
   const templateValues = getTemplateValues(viewName);
 
+  const startBrowsing = () => {
+    const FXA_PRODUCT_PAGE_URL = 'https://www.mozilla.org/firefox/accounts';
+    navigate(FXA_PRODUCT_PAGE_URL, { replace: true });
+  };
+
   return (
     <>
       <CardHeader
         headingText={templateValues.headerText}
         headingTextFtlId={templateValues.headerId}
       />
+      {errorMessage && (
+        <Banner type={BannerType.error}>
+          <p>{errorMessage}</p>
+        </Banner>
+      )}
       <div className="flex justify-center mx-auto">
         <HeartsVerifiedImage className="w-3/5" />
       </div>
       <section>
-        <div className="error"></div>
-        {isSignedIn ? (
-          serviceName ? (
-            <FtlMsg id="ready-use-service" vars={{ serviceName }}>
-              <p className="my-4 text-sm">{`You’re now ready to use ${serviceName}`}</p>
-            </FtlMsg>
-          ) : (
-            <FtlMsg id="ready-use-service-default">
+        {isSync && (
+          <>
+            <FtlMsg id="ready-complete-set-up-instruction">
               <p className="my-4 text-sm">
-                You’re now ready to use account settings
+                Complete set-up by entering the new password on your other
+                Firefox devices.
               </p>
             </FtlMsg>
-          )
-        ) : (
+            <div className="flex justify-center mx-auto mt-6">
+              <FtlMsg id="ready-start-browsing-button">
+                <button className="cta-primary cta-xl" onClick={startBrowsing}>
+                  Start browsing
+                </button>
+              </FtlMsg>
+            </div>
+          </>
+        )}
+        {!isSync && isSignedIn && serviceName && (
+          <FtlMsg id="ready-use-service" vars={{ serviceName }}>
+            <p className="my-4 text-sm">{`You’re now ready to use ${serviceName}`}</p>
+          </FtlMsg>
+        )}
+        {!isSync && isSignedIn && !serviceName && (
+          <FtlMsg id="ready-use-service-default">
+            <p className="my-4 text-sm">
+              You’re now ready to use account settings
+            </p>
+          </FtlMsg>
+        )}
+        {!isSync && !isSignedIn && (
           <FtlMsg id="ready-account-ready">
             <p className="my-4 text-sm">Your account is ready!</p>
           </FtlMsg>
         )}
       </section>
       {continueHandler && (
-        <div className="flex justify-center mx-auto mt-6 max-w-64">
+        <div className="flex justify-center mx-auto mt-6">
```

Comment:
Just curious why did the max-w-64 get removed?

Reply:
To standardize with other CTAs - this was the only button that wasn't full width in the AppLayout.